pandas/tests/extension/json/test_json.py

```diff
@@ -230,3 +230,21 @@ def test_groupby_extension_agg(self, as_index, data_for_grouping):
         super(TestGroupby, self).test_groupby_extension_agg(
             as_index, data_for_grouping
         )
+
+
+def test_ops(data):
```

Comment:
rather than try to create one off tests, you need a genearlized testing mechanism dispatched in BaseOps

Reply:
OK, I copied your code and made some modifications to it to have a general testing mechanism.